### ISSUE TITLE
User can create gene sets with ASCII characters 32-126

### ISSUE THREAD

**Issue by signechambers1** [chanzuckerberg/cellxgene#2096]
User can use ASCII characters 32-126. Allowing all these characters will allow for more flexibility for users and still allow for parsing. This will be consistent with annotations once #2095 is complete.

**Comment by bkmartinjr** [comment]
requires both f/e and b/e changes.

**Comment by signechambers1** [comment]
Closing in favor of https://app.zenhub.com/workspaces/cellxgene-explorer-600f112266e8830017f47e12/issues/chanzuckerberg/cellxgene/2160
